files/default/neutron-ha-tool.py

```diff
@@ -326,6 +326,101 @@ def l3_agent_rebalance(qclient, noop=False, wait_for_router=True):
     :param qclient: A neutronclient
     :param noop: Optional noop flag
     """
+    agent_list = live_agent_list(qclient)
+    router_mover = RouterMover(qclient, noop, wait_for_router)
+
+    while agent_list.calculate_maximal_router_count_difference() > 1:
```

Comment:
Much better, thanks!  In fact, I'd now describe this commit as beautifully clear and clean code :-)